index.html

```diff
@@ -394,17 +394,17 @@ <h6 data-lang="zh-hant">發表對本文檔的建議</h6>
             <p data-lang="zh-hant">為使字數不同的標題間能取得平衡，而加大字距。</p>
           </li>
           <li>
-            <p data-lang="en">For captions of illustrations and tables, which only have a few characters, increased inter-character spacing is used to balance with the size of an illustration or table.</p>
+            <p data-lang="en">For captions of illustrations and tables, which only have a few characters, increased inter-character spacing is used to achieve balance with the size of the illustration or table.</p>
             <p data-lang="zh-hans">图片与表格之说明文字字数较少时，为取得平衡，而加大字距。</p>
             <p data-lang="zh-hant">圖片與表格之說明文字字數较少時，為取得平衡，而加大字距。</p>
           </li>
           <li>
-            <p data-lang="en">In some cases, increased inter-character spacing is used for poetry where one line has only a few characters, so as to maintain the balance of the layout.</p>
+            <p data-lang="en">In some cases, increased inter-character spacing is used for poetry, where one line has only a few characters, so as to maintain the balance of the layout.</p>
```

Comment:
The comma changes the meaning. Previously it implied that spacing is used only for lines in a poem that have a small number of characters.   Now it implies that, because lines in poems have a small number of characters in general spacing is applied liberally to lines in poems.  Which is correct?